src/hooks/__internal__/useMenuKeyboardNavigation/useMenuKeyboardNavigation.ts

```diff
@@ -11,7 +11,10 @@ export default (
 
   const handleKeyDown = useCallback(
     (ev) => {
-      ev.preventDefault();
+      if (!(Events.isEnterKey(ev) || Events.isSpaceKey(ev))) {
```

Comment:
While this is probably OK because I can't think of a specific case it would break anything, I think it's probably safer to only explicitly call `ev.preventDefault()` in the cases where we know we specifically want it (the arrow keys), rather than blanket call it whenever the key isn't Enter or Space. But as I said this is probably OK, and can easily be updated if we discover edge cases I guess.

Reply:
Yeah I had the same thought, I decided to go with the above as we also need to prevent the default for some of the tabbing behaviour (when leaving the child button container) so went with this to avoid adding it in every if/ else block etc. For now I'd prefer to leave it as is and we can update if we find any edge cases.